examples/resources/cluster/resource.tf

```diff
@@ -1,27 +1,42 @@
-resource "risingwavecloud_cluster" "example" {
-  name    = "dev"
-  version = "v1.3.0"
-  region  = "us-east-2"
-  resourcev1 = {
+resource "risingwavecloud_cluster" "my_cluster" {
+  region  = "us-east-1"
+  name    = "my_cluster"
+  version = "v1.8.0"
+  spec = {
     compute = {
-      type    = "p-1c2g"
-      replica = 1
+      default_node_group = {
```

Comment:
Question here: can there be more node groups?

Reply:
yes, extra node group will be supported in the later version as an attribute or a resource like `cluster_node_group`.  We can discuss the details later.